pkg/gce-pd-csi-driver/controller.go

```diff
@@ -1327,3 +1351,95 @@ func pickRandAndConsecutive(slice []string, n int) ([]string, error) {
 	}
 	return ret, nil
 }
+
+func (gceCS GCEControllerServer) CheckAndUpdateLastKnownInstanceOps(ctx context.Context, project, location, instanceName string) error {
+	ops := gceCS.opsCache.InstanceOps.GetOps(common.CreateInstanceKey(project, location, instanceName))
+	if ops == nil {
+		return nil
+	}
+
+	for _, op := range ops {
+		done, err := gceCS.CloudProvider.CheckOpDoneStatus(ctx, project, location, op.Name)
```

Comment:
should we consider using `op.Id` instead? [api](https://cloud.google.com/compute/docs/reference/rest/v1/zoneOperations) specs `Id` to be the unique identifier

Reply:
The existing code seems to use op.name instead of the id.
https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/master/pkg/gce-cloud-provider/compute/gce-compute.go#L520